### ISSUE TITLE
Plugin Request: Hide Channels

### ISSUE THREAD

**Issue by Venomousicon** [Vencord/plugin-requests#669]
### Plugin Description

This adds a button at the top left in the tool bar to click which hides the channel list.

Additionally there is also a keybind option (default: Ctrl + H) you can use and rebind.
This is really useful, especially when you resize the Discord window to a smaller size and the channel list takes up half the screen
![124384178-9618bd00-dcbf-11eb-970e-f78d8093dafc](https://github.com/Vencord/plugin-requests/assets/98240235/f16b5388-b4bb-40b6-9e3d-9756df9f74f0)

### Existing Plugin for other mod

https://betterdiscord.app/plugin/Hide%20Channels

### Request Agreement

- [X] I read the above and made sure my plugin is possible and allowed
- [X] I checked the [existing plugins](https://vencord.dev/plugins) and made sure my plugin doesn't already exist
- [X] I [searched the existing issues](https://github.com/Vencord/plugin-requests/issues?q=is%3Aissue) and made sure a similar request doesn't already exist

**Comment by Venomousicon** [comment]
I'm not sure if something like this already exists in Vencord, I thought it did but I checked earlier and didn't seem to find anything like it in the plugins list.

**Comment by programminglaboratorys** [comment]
> I'm not sure if something like this already exists in Vencord, I thought it did but I checked earlier and didn't seem to find anything like it in the plugins list.

you may saw a css snippet

**Comment by Scyye** [comment]
9if it can be done with a css snippet, then whats the reason for making it into a plugin?

**Comment by rozbrajaczpoziomow** [comment]
I don't think you can fully do that with css :3, especially given if you want a keybind or button.

**Comment by Venomousicon** [comment]
> 9if it can be done with a css snippet, then whats the reason for making it into a plugin?

Because you can't just do it with a CSS snippet if you want a keybind or button to open and close it as you please.
